File: C/FlowControl/ifelse.c
#include <stdio.h>

int main() {
    int number;
    printf("Enter an integer: ");
    scanf("%d", &number);

    // true if number is less than 0
    if (number < 0) {
        printf("You entered %d and is less than zero.\n", number);
    }
    // false if number is less than 0
    else
        printf("You entered %d and is greater than or equal to zero.\n", number);
    printf("The if statement is easy.\n");

    return 0;
}

File: Python/FlowControl/ifelse.py
def main():
    number = int(input("Enter an integer: "))

    # true if number is less than 0
    if number < 0:
        print(f"You entered {number} and it is less than zero.")
    # false if number is less than 0
    else:
        print(f"You entered {number} and it is greater than or equal to zero.")
    
    print("The if statement is easy.")

if __name__ == "__main__":
    main()